packages/theming/atlas/src/themesource/atlas_core/native/core/helpers/radiobuttons.ts

```diff
@@ -0,0 +1,103 @@
+/*
+
+DISCLAIMER:
+Do not change this file because it is core styling.
+Customizing core files will make updating Atlas much more difficult in the future.
+To customize any core styling, copy the part you want to customize to styles/native/app/ so the core styling is overwritten.
+==========================================================================
+*/
+
+import { brand } from "../../../../../theme/native/custom-variables";
+import { RadioButtonsStyle } from "../../types/widgets";
+
+// Radio buttons properties
+export const radioButtonsPrimary: RadioButtonsStyle = {
```

Comment:
Would it be better to create two interfaces, one for the label and one for the button? Maybe this could even help during modeler design time in page editor.

Reply:
Something like 

```
interface RadioButtonsLabelStyle {
    labelTextStyle?: TextStyle;
}

interface RadioButtonsButtonAndTitleStyle {
    circularButtonStyle?: ViewStyle;
    activeButtonStyle?: ViewStyle;
    radioButtonItemTitleStyle?: TextStyle;
}

interface RadioButtonsStyle {
    radioButtonItemContainerStyle?: ViewStyle;
    radioButtonItemContainerHorizontalStyle?: ViewStyle;
    validationMessage?: TextStyle;
} & RadioButtonsButtonAndTitleStyle & RadioButtonsLabelStyle
```